source/guides/mattermost-ai-copilot-admin-guide.rst

```diff
@@ -0,0 +1,74 @@
+Admin Guide: Mattermost AI Copilot Plugin
+=========================================
+
+This document provides instructions for administrators to configure `Mattermost AI Copilot <https://github.com/mattermost/mattermost-plugin-ai>`_. For the corresponding user guide, refer to the `Mattermost AI Copilot end user documentation <../guides/mattermost-ai-copilot-user-guide.html>`_.
+
+For the best experience with Mattermost AI Copilot, we recommend using **Mattermost Server v9.6** or later. Mattermost AI Copilot is installed as a plugin and is compatible with the following `Mattermost server versions <https://docs.mattermost.com/deploy/mattermost-changelog.html>`_:
+
+- **v9.6** or later
+- **v9.5.2+** (Extended Support Release - ESR)
+- **v9.4.4+**
+- **v9.3.3+**
+- **v8.1.11+** (Extended Support Release - ESR)
+
+Configuration Steps
+-------------------
+
+Configure the plugin via **System Console ➡️ Mattermost AI Copilot Plugin**. The following AI services are supported:
```

Comment:
To clarify my earlier comment, I think we still want to first mention that admins have to install the plugin if <v9.7 with a link to the releases page of the repo. If >= v9.7 then they can follow the steps written here.

Reply:
Okay, reworked this intro to better incorporate all of this.